class-instant-articles-post.php

```diff
@@ -991,6 +991,10 @@ public function set_appearance_from_settings() {
 			$this->instant_article->withStyle( 'default' );
 		}
 
+		if (has_filter('instant_articles_style')) {
```

Comment:
Its better to remove the method `get_article_style()` and call inline the `apply_filters`.

Also add a documentation block before this method call `apply_filters`
```
                /**
		 * Apply the default filter 'instant_articles_style' for the instant article.
		 *
		 * @since 4.0.6
                 * @param string                           $template                     Path to the current (default) template.
		 * @param Instant_Article_Post    $instant_article_post   The instant article post.
		 */
                 $article_style = apply_filters( 'instant_articles_style', 'default', $this );
                  if ( $article_style !== null && !Type::isTextEmpty( $article_style ) ) {
                      $instant_article->withStyle( $article_style );
                  }
```